src/Modules/StatCan.OrchardCore.Assessment/Recipes/Assessment.recipe.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "Assessment",
+    "displayName": "Assessment",
+    "description": "Creates an Assessment tool. It requires a JSON file generated through https://surveyjs.io/ ",
+    "author": "StatCan Digital Innovation",
+    "website": "digital.statcan.gc.ca",
+    "version": "1.0.0",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [], 
+    "steps": [
+        {
+            "name": "feature",
+            "disable": [],
+            "enable": [
+                "StatCan.OrchardCore.Assessment"
+            ]
+        },
+        {
+            "name": "ContentDefinition",
+            "ContentTypes": [
+              {
+                "Name": "Assessment",
```

Comment:
This content type creation can be moved to `Migrations.cs` like was done for Tabulator/TableCreator.